controllers/contacts.js

```diff
@@ -0,0 +1,11 @@
+const models = require('../models')
+
+const getAllCustomerInfo = async (request, response) => {
+  const customers = await models.Customers.findAll()
+
+  return response.send(customers)
+}
```

Comment:
Create a new functions that will handle
Deleting a customer.
Create a customer.
Update a customer.